Make.versions

```diff
@@ -48,8 +48,8 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=17.5.0.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=9.5.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=17.99.0.$(IOS_COMMIT_DISTANCE)
+MAC_PACKAGE_VERSION=9.99.0.$(MAC_COMMIT_DISTANCE)
```

Comment:
Not that it really matters but… 😃 

```suggestion
IOS_PACKAGE_VERSION=18.0.0.$(IOS_COMMIT_DISTANCE)
MAC_PACKAGE_VERSION=10.0.0.$(MAC_COMMIT_DISTANCE)
```

Reply:
This will all be removed from main eventually :smile: